pkg/controller/podpreset/admission.go

```diff
@@ -0,0 +1,355 @@
+/*
```

Comment:
I don't think so. The other admission controllers are for the main controller while this admission controller is for the pod preset controller.

Reply:
It's not an admission controller, per se - it's a controller that implements an initializer. Fine semantic difference, but this should not go under `plugin/pkg`

Reply:
Needs to be called "initializer" or something, and the `admit` should become `initialize`. I don't want there to be any cross talk of idea/names since they are so different in their lifecycle.